tests/test_csc.py

```diff
@@ -111,11 +129,36 @@ async def test_stop_propagate_laser(self):
                 topic=self.remote.evt_detailedState,
                 detailedState=TunableLaser.LaserDetailedState.NONPROPAGATING,
             )
+            with pytest.raises(salobj.AckError):
+                await self.remote.cmd_stopPropagateLaser.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_startPropagateLaser.set_start(timeout=STD_TIMEOUT)
```

Comment:
Please check that this actually makes the laser propagate, e.g. by checking self.remote.evt_detailedState or something.